shell/client/shell.html

```diff
@@ -166,13 +167,27 @@
         {{else}}
           {{#if selectedTab.myFiles}}
             <div class="button-box">
-              <button id="install-apps-button">Install apps</button>
-              <button id="upload-app-button">Upload an app</button>
-              <button id="restore-backup-button">Restore a backup</button>
+              <button id="install-apps-button" class="{{#if overQuota}}over-quota{{/if}}">
+                  Install apps</button>
+              <button id="upload-app-button" class="{{#if overQuota}}over-quota{{/if}}">
+                  Upload an app</button>
+              <button id="restore-backup-button" class="{{#if overQuota}}over-quota{{/if}}">
+                  Restore a backup</button>
             </div>
           {{/if}}
         {{/if}}
 
+        {{#unless selectedTab.sharedWithMe}}
+          {{#if quotaEnabled}}
+            <div id="storage-usage" class="{{#if overQuota}}over-quota{{/if}}">{{storageUsage}}
```

Comment:
Yeah I consider this provisional until the UI redesign kicks in. FWIW, gmail shows storage usage at the bottom of every screen, even when you're under a label or something that doesn't include all your mail. (In fact, I think the storage quota it shows is shared with Google Drive, even.)